.github/workflows/release-please.yml

```diff
@@ -15,17 +15,25 @@ jobs:
     outputs:
       releases_created: ${{ steps.release.outputs.releases_created }}
     steps:
-      - uses: google-github-actions/release-please-action@v4
-        id: release
+      - uses: google-github-actions/release-please-action@v3
         with:
-          # this assumes that you have created a personal access token (PAT)
+          command: manifest
           token: ${{ secrets.LEATHER_BOT }}
-          # optional. customize path to release-please-config.json
-          config-file: release-please-config.json
-          # optional. customize path to .release-please-manifest.json
-          manifest-file: .release-please-manifest.json
-          # Our target branch should be `dev`
-          target-branch: dev
+          default-branch: dev
+
+    # TODO: get this back when release-please-action@4 gets an update for pnpm node workspaces.
```

Comment:
Is there a task we can link in the commit message? Or else add here so we don't forget about this in a sea of TODOs